src/OrchardCore/OrchardCore.ContentManagement.Display/IContentItemDisplayManager.cs

```diff
@@ -11,7 +11,7 @@ namespace OrchardCore.ContentManagement.Display
     public interface IContentItemDisplayManager
     {
         Task<IShape> BuildDisplayAsync(ContentItem content, IUpdateModel updater, string displayType = "", string groupId = "");
-        Task<IShape> BuildEditorAsync(ContentItem content, IUpdateModel updater, string groupId = "", string htmlFieldPrefix = "");
-        Task<IShape> UpdateEditorAsync(ContentItem content, IUpdateModel updater, string groupId = "", string htmlFieldPrefix = "");
+        Task<IShape> BuildEditorAsync(ContentItem content, IUpdateModel updater, bool isNew, string groupId = "", string htmlFieldPrefix = "");
+        Task<IShape> UpdateEditorAsync(ContentItem content, IUpdateModel updater, bool isNew, string groupId = "", string htmlFieldPrefix = "");
     }
```

Comment:
So, here there is no default value for `IsNew`. Maybe this is with this one that we would have more cases where we would not need to pass this parameter.

But as i said in my last comment for `IDisplayManager<TModel>`, to prevent some errors maybe better to not use a default value for `IsNew`.